From 4042b3281152f2f5ba16fe4c76a602a4d6131f56 Mon Sep 17 00:00:00 2001
From: Adrian Alonso <aalonso@freescale.com>
Date: Tue, 10 Nov 2015 17:42:29 -0600
Subject: [PATCH 07/37] MLK-11888-1: arm: imx6: common: fix ddr_type
 identification

Fix run time ddr type identification as function that
access MMDC register is not enabled due to incorrect macro
name, fix config macro names to enable functions
imx_mmdc_get_ddr_type and imx_ddrc_get_ddr_type

Signed-off-by: Adrian Alonso <aalonso@freescale.com>
Signed-off-by: Anson Huang <b20788@freescale.com>
---
 arch/arm/mach-imx/common.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm/mach-imx/common.h b/arch/arm/mach-imx/common.h
index cb3873f..71e4d53 100644
--- a/arch/arm/mach-imx/common.h
+++ b/arch/arm/mach-imx/common.h
@@ -181,12 +181,12 @@ int imx6q_set_lpm(enum mxc_cpu_pwr_mode mode);
 void imx6q_set_int_mem_clk_lpm(bool enable);
 void imx6sl_set_wait_clk(bool enter);
 void imx6_enet_mac_init(const char *compatible);
-#ifdef CONFIG_HAVE_MMDC
+#ifdef CONFIG_HAVE_IMX_MMDC
 int imx_mmdc_get_ddr_type(void);
 #else
 static inline int imx_mmdc_get_ddr_type(void) { return 0; }
 #endif
-#ifdef CONFIG_HAVE_DDRC
+#ifdef CONFIG_HAVE_IMX_DDRC
 int imx_ddrc_get_ddr_type(void);
 #else
 static inline int imx_ddrc_get_ddr_type(void) { return 0; }
-- 
1.9.1

